# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1763631343 -3600
# Parent  eee249044db7dbd188b49997ab71e888974ba89b
Misc: Hide QF search button if gloda is disabled.

diff --git a/mail/themes/shared/mail/search-bar.css b/mail/themes/shared/mail/search-bar.css
--- a/mail/themes/shared/mail/search-bar.css
+++ b/mail/themes/shared/mail/search-bar.css
@@ -70,8 +70,17 @@ input {
     padding: 2px;
   }
 }
 
+@media not -moz-pref("mailnews.database.global.indexer.enabled") {
+  #search-button {
+    display: none;
+  }
+  #clear-button {
+    inset-inline-end: 0 !important;
+  }
+}
+
 div {
   display: none;
   position: absolute;
   pointer-events: none;
